src/Microsoft.AspNet.Mvc.TagHelpers/Gruntfile.js

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+module.exports = function (grunt) {
+    grunt.initConfig({
+        jshint: {
+            scripts: [ "js/**/*.js" ]
+        },
+        uglify: {
+            scripts: {
+                files: [{
+                    expand: true,
+                    cwd: "js",
+                    src: "**/*.js",
+                    dest: "Compiler/Resources"
```

Comment:
you're loading .js files into the TagHelpers assembly?  why not place LinkTagHelper_FallbackJavaScript.js into this directory to start with?  using grunt here seems like overkill.